core/scripts/cli/sponsor/showMarketDetails.js

```diff
@@ -28,6 +33,36 @@ const showMarketDetails = async (web3, artifacts, emp) => {
       const isWeth = await getIsWeth(web3, artifacts, collateralCurrency);
       const collateralSymbol = await getCurrencySymbol(web3, artifacts, collateralCurrency);
 
+      // TODO: potentially generalize this for use elsewhere in the CLI tool.
+      const getCollateralizationRatio = async () => {
+        let priceFeed;
+        try {
+          priceFeed = await createReferencePriceFeedForEmp(
+            winston.createLogger({ silent: true }),
+            web3,
+            new Networker(),
+            () => Math.floor(Date.now() / 1000),
+            emp.address
+          );
+        } catch (error) {
+          // Ignore error
+        }
+
+        if (!priceFeed) {
+          return "Unknown";
+        }
+
+        await priceFeed.update();
+        const collateralizationRatio = await computeCollateralizationRatio(
+          web3,
+          priceFeed.getCurrentPrice(),
```

Comment:
If the pricefeed is constructed properly, should we expect that `getCurrentPrice()` should occasionally fail and return `null`? If so, then we should handle that either before calling `computeCollateralizationRatio` or in that method.

Reply:
Good point. Will handle that case.

Reply:
Fixed.